NYTPhotoViewer/NYTPhotosViewController.m

```diff
@@ -235,7 +235,7 @@ - (void)addOverlayView {
     
     [self updateOverlayInformation];
     [self.view addSubview:self.overlayView];
-    
+
```

Comment:
@benfrye Was the additional space here intentional?

Reply:
I think this is removing spacing that my linter yelled about.